src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -3194,7 +3194,7 @@ def create_vmss(cmd, vmss_name, resource_group_name, image=None,
                 user_data=None, network_api_version=None, enable_spot_restore=None, spot_restore_timeout=None,
                 capacity_reservation_group=None, enable_auto_update=None, patch_mode=None, enable_agent=None,
                 security_type=None, enable_secure_boot=None, enable_vtpm=None, automatic_repairs_action=None,
-                v_cpus_available=None, v_cpus_per_core=None, accept_term=None, disable_integrity_monitoring=False,
+                v_cpus_available=None, v_cpus_per_core=None, accept_term=None, enable_integrity_monitoring=False,
```

Comment:
```suggestion
                v_cpus_available=None, v_cpus_per_core=None, accept_term=None, disable_integrity_monitoring=False, enable_integrity_monitoring=False,
```

Reply:
Keeping `disable_integrity_monitoring=False` will no longer have any effect, just to avoid unexpected breaking changes to customers

Reply:
Done